symforce/slam/imu_preintegration/preintegrated_imu_measurements.h

```diff
@@ -0,0 +1,63 @@
+/* ----------------------------------------------------------------------------
+ * SymForce - Copyright 2022, Skydio, Inc.
+ * This source code is under the Apache 2.0 license found in the LICENSE file.
+ * ---------------------------------------------------------------------------- */
+
+#pragma once
+
+#include <Eigen/Core>
+
+#include <sym/rot3.h>
+
+namespace sym {
+
+/**
+ * Struct of Preintegrated IMU Measurements (not including the covariance of change in
+ * orientation, velocity, and position).
+ */
+template <typename Scalar>
+struct PreintegratedImuMeasurements {
+  using Vector3 = Eigen::Matrix<Scalar, 3, 1>;
+  using Matrix33 = Eigen::Matrix<Scalar, 3, 3>;
+
+  // The rotation that occured over the measurement period; i.e., maps the coordinates of a vector
+  // in the body frame of the end of the measurement period to the coordinates of the vector in the
+  // body frame at the start of the measurement period.
+  sym::Rot3<Scalar> DR;
+
+  // The velocity change that occured over the measurement period in the body frame of the
+  // initial measurement (assuming 0 acceleration due to gravity)
+  Vector3 Dv;
+
+  // The position change that occured over the measurement period in the body frame of the
+  // initial measurement (assuming 0 acceleration due to gravity)
+  Vector3 Dp;
```

Comment:
This is also assuming 0 initial velocity, right?

Reply:
Ope. Yup. Though I do have a slight worry that this might not be the best way to phrase it. As it's not so much that `Dp` depends on acceleration and initial velocity being 0 as in the interpretation as being the position change depends on that. Really, it's the position change with the effects of gravity and initial velocity subtracted out.

But I won't sweat it too much and just amend parenthetical statement to mention initial velocity is also assumed to be 0.